recipes/terraform-provider-kubectl/meta.yaml

```diff
@@ -20,10 +20,7 @@ build:
   number: 0
   script: |
     go build -v -o {{ target_gobin }}..{{ os.sep }}{{ plugin_dir }}{{ os.sep }}{{ target_goos }}_{{ target_goarch }}{{ os.sep }}{{ x4_plugin }}{{ target_goexe }}
-    # save thirdparty licenses
-    go-licenses save . --save_path ./thirdparty
-    # Clear out cache to avoid file not removable warnings
-    chmod -R u+w $(go env GOPATH) && rm -r $(go env GOPATH)  # [not win]
```

Comment:
```
chmod -R u+w $(go env GOPATH) && rm -r $(go env GOPATH)  # [not win]
```
I'd be inclined to keep this. it only shaves a few seconds off the build *time* but it removes ~20k of junk output in the logs.